api/src/db/DynamoBusinessesDataClient.test.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
+import { dynamoDbTranslateConfig } from "@db/config/dynamoDbConfig";
+import { DynamoBusinessesDataClient } from "@db/DynamoBusinessesDataClient";
+import { BusinessesDataClient } from "@domain/types";
+import { generateBusiness, generateProfileData, generateTaxFilingData } from "@shared/test";
+import dayjs from "dayjs";
+
+// references jest-dynalite-config values
+const dbConfig = {
+  tableName: "businesses-table-test",
+};
+
+describe("DynamoBusinessesDataClient", () => {
+  const config = {
+    endpoint: process.env.MOCK_DYNAMODB_ENDPOINT,
+    sslEnabled: false,
+    region: "local",
+  };
+
+  let client: DynamoDBDocumentClient;
+  let dynamoBusinessesDataClient: BusinessesDataClient;
+
+  beforeEach(() => {
+    client = DynamoDBDocumentClient.from(new DynamoDBClient(config), dynamoDbTranslateConfig);
+    dynamoBusinessesDataClient = DynamoBusinessesDataClient(client, dbConfig.tableName);
+  });
+
+  it("Test to verify db gets inserted items", async () => {
+    await expect(dynamoBusinessesDataClient.get("some-id")).rejects.toEqual(new Error("Not found"));
+    const businessesData = generateBusiness({
+      profileData: generateProfileData({
+        taxId: "123456789000",
+      }),
+    });
+    const businessId = businessesData.id;
+    await dynamoBusinessesDataClient.put(businessesData);
+
+    const expectedValue = await dynamoBusinessesDataClient.get(businessId);
+    const { version, ...filteredExpectedValue } = expectedValue as unknown as {
+      version?: string;
+      [key: string]: any;
+    };
```

Comment:
Huh. I didn't realize you could use the spread syntax during destructuring like this or chain type assertions. Clever!

Reply:
Yup, that’s exactly what’s happening. Glad you found it interesting! The spread syntax during destructuring can be pretty handy, and chaining type assertions can help streamline things. Always learning new tricks with TS